requirements/pip.txt

```diff
@@ -4,11 +4,11 @@
 #
 #    make upgrade
 #
-wheel==0.37.0
+wheel==0.37.1
     # via -r requirements/pip.in
 
 # The following packages are considered to be unsafe in a requirements file:
 pip==21.3.1
     # via -r requirements/pip.in
-setuptools==59.5.0
+setuptools==60.1.0
```

Comment:
@UsamaSadiq @aht007 I thought we added a constraint  `setuptools<60`

Reply:
`pip.in` didn't have common constraints included in it. I'll update the file to include common constraints in it.